voter-dapp/src/RetrieveRewards.js

```diff
@@ -117,7 +117,7 @@ function RetrieveRewards({ votingAccount }) {
       return MAX_UINT_VAL;
     } else {
       // This section will produce an array of roundIds that should be queried for unclaimed rewards.
-      const defaultLookback = 10; // Default lookback is 10 rounds.
+      const defaultLookback = 7; // Default lookback is 7 rounds (2 weeks).
```

Comment:
OOC what was your thinking behind this hard coded value?

Reply:
Rewards expire after two weeks (7 voting periods). I can add a comment.

Reply:
Makes sense